user/caching.md

```diff
@@ -135,7 +135,16 @@ podfile: path/to/Podfile
 
 ### npm cache
 
-For caching with `npm`, use:
+> Please note that npm is now cached by default on Travis CI
+
+To disable npm caching, use:
+
+```yaml
+cache:
+  npm: false
+```
+
+To explicitly cache `npm`, use:
```

Comment:
Is it worth deleting this line and the following block? Or is there a use case for explicitly caching npm?

Reply:
I think we can have this explicit statement here. It is unlikely, but we _could_ switch the default behavior back to not caching, in which case the explicit `cache` will save the users.

Reply:
Just had a chat with @BanzaiMan, we will leave it since it might be useful for some users 

From slack: 
>hasari 12:09 PM I think leaving it there is fine; doing things implicitly can be a trap, when that default changes for some reason again.
> (relying on Ruby being the default, Precise/Trusty/Xenial/Bionic being default, etc., etc.)
> when the default can (or did) change, the being explicit is a good thing.